assets/css/_skate_ui.scss

```diff
@@ -0,0 +1,115 @@
+// #region Text Styles [v2]
+
+// https://www.notion.so/mbta-downtown-crossing/2022-12-19-Type-Styles-6f6c90eb449248b7a954e234b718137f
```

Comment:
**suggestion:** I think it would be worth running this by design to see if this matches their general approach to things. Overall I really like these changes, though.

Reply:
Ideally we could eventually get rid of some of the type weight / size stuff in `_ui_kit.scss` as well.

Reply:
I can't tell if you're referring to the implementation (separating out font-weight/size from color) or the document(which was made by design :D), but I'll definitely make sure I sync with design before merging.

Reply:
🤦 I totally didn't realize that design created that Notion document. In that case I'd say this particular change is good to go!